ansible_collections/arista/avd/roles/eos_designs/templates/underlay/interfaces/ethernet-interfaces.j2

```diff
@@ -33,7 +33,7 @@ ethernet_interfaces:
     ospf_area: {{ underlay_ospf_area }}
 {%         endif %}
 {%         if switch.underlay_routing_protocol == "isis" %}
```

Comment:
so all the new protocols will _only_ work with `backbone interfaces`. Not the `uplink_interfaces`?

Reply:
Yes, i focused on backbone only because i think the target networks for this design won't really be leaf-spine topologies, atleast that's the feedback i've gotten from the SP working group. I have nothing against adding support for them in uplink_interfaces per se, maybe we can add it after the main PR is merged. I'll be sure to add it to the documentation.